scripts/qa_crowbarsetup.sh

```diff
@@ -5302,6 +5321,10 @@ function onadmin_crowbar_nodeupgrade
         fi
 
         if [[ $want_nodesupgrade ]]; then
+            local upgrade_mode=`safely crowbarctl upgrade status --format plain --filter=suggested_upgrade_mode`
```

Comment:
Also, `suggested_upgrade_mode` could be changed by user, then `selected_upgrade_mode` applies. Not happening for mkcloud deployment, but still probably better to just call `crowbarctl upgrade mode`